src/components/Tables/ManagersTable/ManagersTable.const.tsx

```diff
@@ -32,10 +32,23 @@ export const getHeadCells = (): HeadCell[] => [
   },
 ];
 
-export const getActions = ({ editAccessAction }: Actions) => [
-  {
-    icon: <Svg id="edit-user" />,
-    action: editAccessAction,
-    toolTipTitle: t('editAccess'),
-  },
+export const getActions = (
+  isOwner: boolean,
+  id: string | undefined,
+  { removeAccessAction, editAccessAction }: Actions,
+) => [
+  ...(isOwner || id
+    ? []
```

Comment:
why nested array

Reply:
not nested. spreaded

Reply:
do just like this isOwner || id ? [] : [....]